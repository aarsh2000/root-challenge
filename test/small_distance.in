Driver small
Trip small 00:00 00:01 0.01