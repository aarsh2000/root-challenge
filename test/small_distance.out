small: 0 miles
